DeploymentCloud/Deployment.Common/Resources/Parameters/Spark-Parameter.json

```diff
@@ -25,7 +25,10 @@
 			}
 		},
 		"sparkClusterVersion": {
-			"value": "3.6"
+			"value": "$HDInsightVersion"
+		},
+		"sparkComponentVersion": {
+			"value": "$sparkComponentVersion"
```

Comment:
Should we add "minNodesForHDInsightAutoScaling" and "maxNodesForHDInsightAutoScaling" paramerters to this file ?

Reply:
This should be the same issue as we discussed in another comment. If we decide to use one single parameter file for the two templates, yes, we need to add the two autoscaling related parameters to this file.

Reply:
added the two parameters to the Spark-Parameter.json file